spec/unit/render_spec.rb

```diff
@@ -0,0 +1,20 @@
+RSpec.describe TTY::ProgressBar, '#render' do
```

Comment:
Adding this means tests break because the String passed to StringIO.new is frozen.

Adding the `+""` means tests break on ruby versions that don't have String#+@.

No other test files have the frozen string literal comment so I'm leaving it off.